### ISSUE TITLE
Backend login "Authentication failed" on new install of fusio 2.0.3 - missing /backend/token route?

### ISSUE THREAD

**Issue by mv-steven** [apioo/fusio#395]
Hello,
I believe my system is setup correctly.  I downloaded the release zip file for 2.0.3, ran `public/install.php` via the browser to configure my database & user, then ran `php bin/fusio marketplace:install fusio` to install the backend UI.  I have mod_rewrite turned on, etc.

The login screen at `public/apps/fusio/#!/login` is POSTing its authentication request to `public/index.php/backend/token`, but the request is failing and the UI displays "Authentication failed".  Running via the debugger I see that the `resolve()` function pulls all routes from the `fusio_routes` table for `category_id = 2` at `src/Framework/Loader/LocationFinder/DatabaseFinder.php:73`, but none of the `fusio_routes.path` values match "/backend/token", so `resolve()` returns null, which triggers `load()` to throw an `InvalidPathException` at `src/Loader/Loader.php:104`.

Here's what's in my `fusio_routes` table matching '%token%':

```
MariaDB [fusio]> select * from fusio_routes where path like '%token%' and category_id = 2;
+----+-------------+--------+----------+---------+----------------------------------------------+-------------------------------------------+
| id | category_id | status | priority | methods | path                                         | controller                                |
+----+-------------+--------+----------+---------+----------------------------------------------+-------------------------------------------+
| 14 |           2 |      1 |     2007 | ANY     | /backend/app/token                           | Fusio\Impl\Controller\SchemaApiController |
| 15 |           2 |      1 |     2008 | ANY     | /backend/app/token/$token_id<[0-9]+>         | Fusio\Impl\Controller\SchemaApiController |
| 18 |           2 |      1 |     2011 | ANY     | /backend/app/$app_id<[0-9]+>/token/:token_id | Fusio\Impl\Controller\SchemaApiController |
| 68 |           2 |      1 |     2061 | ANY     | /backend/statistic/issued_tokens             | Fusio\Impl\Controller\SchemaApiController |
+----+-------------+--------+----------+---------+----------------------------------------------+-------------------------------------------+
```

I'm not sure what next step to take.  I tried adding a "/backend/token" path manually with the same data as row 14 above, but then the application returns an http 415 error because an `Accept` header value it's looking for isn't present.

Did I miss a step that would have setup the "/backend/token" route?  Or is something else going on?

**Comment by chriskapp** [comment]
Hi, so with the last release we have consolidated our authorization routes, so the frontend app should make a request to the `/token/authorization` endpoint, we have removed the `/backend/token` and `/consumer/token` endpoints, the latest version of the backend should also use this route, could you please check whether you are using the latest version of the backend app.

**Comment by mv-steven** [comment]
I think I figured out what was going on. I initially installed via composer, couldn't get the backend app installed via `bin/fusio marketplace:install fusio`, so I downloaded the backend app from https://www.fusio-project.org/marketplace and unzipped it in the apps directory.  I was able to see the login screen but the login was still failing due to the class not found error.

I reinstalled fusio from zip then was able to get `bin/fusio marketplace:install fusio` to work.  I think what happened is my browser cached the login page from my initial manual marketplace download (which is version 0.6.0), and it was trying to send a request to the old `/backend/token` endpoint.  I think my browser "un-cached" the page over the weekend; when I try now, it works fine.

I didn't take time to document what was going wrong with the initial install via composer, I may have just overlooked a step.

Thanks for the help!
